# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/fetch/InternalRequest.cpp
# Commit: c0cb3c17f246
# Full Hash: c0cb3c17f2461a19203f3b295a6c56b417483b2a
# Author: Eden Chuang <echuang@mozilla.com>
# Date: 2023-01-12 21:30:33
# Regressor Bug: 1351231
# File Overlap Count: 2
# Description:
#   Bug 1351231 - FetchService integration for PFetch. r=dom-worker-reviewers,jesup
#   
#   Depends on D142436
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D142437
# ==============================================================================

diff -r 84162d09eef8 -r c0cb3c17f246 dom/fetch/InternalRequest.cpp
--- a/dom/fetch/InternalRequest.cpp	Thu Jan 12 15:09:45 2023 +0000
+++ b/dom/fetch/InternalRequest.cpp	Thu Jan 12 15:09:45 2023 +0000
@@ -193,7 +193,7 @@
   if (!XRE_IsParentProcess()) {
     if (body) {
       MOZ_ASSERT(body->type() == BodyStreamVariant::TParentToChildStream);
-      mBodyStream = body->get_ParentToChildStream().stream();
+      mBodyStream = body->get_ParentToChildStream().get_RemoteLazyInputStream();
     }
   } else {
     if (body) {